draft-ietf-rtcweb-jsep.xml

```diff
@@ -3265,7 +3265,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             stored.</t>
           </list></t>
 
-          <t>As required by
+          <t>This list is non-exhaustive and implementations SHOULD
```

Comment:
"This list is non-exhaustive" makes it seem like we may have overlooked something. I would write this as:

> Other attributes that are not relevant to JSEP may also be present, and implementations SHOULD process any that they recognize. As required by...

Note also that this text should also be included in the handling of media-level attrs below (line 3436).